### PR TITLE
fix #48: add syncInfo endpoint

### LINKED ISSUE #48: `syncInfo` query to return synchronizer sync block and global blocknum
@chrisli30 FYI
```
{
  'syncBlockNum': Int!
  'chainBlockNum': Int!
}
```

### PR DESCRIPTION
**Pull request by frankobe**

### CODE REVIEW

File: src/schema/resolvers.js [hunk 1]
```diff
@@ -172,6 +172,35 @@ module.exports = {
...
+        blocks = await Blocks.find({}, options).toArray();
+      } catch(err){
+        console.error(`Error query latest block from db: ${err.message}`);
+      }
+
+      if(blocks.length > 0){
+        syncBlockNum = blocks[0].blockNum;
+      }
+
+      let chainBlockNum = null;
```
Comment (chrisli30) [inline src/schema/resolvers.js]: Define all variables at the beginning of a function.

File: src/schema/resolvers.js [hunk 2]
```diff
@@ -172,6 +172,35 @@ module.exports = {
...
+      try {
+       let resp = await fetch('https://testnet.qtum.org/insight-api/status?q=getInfo');
```
Comment (chrisli30) [inline src/schema/resolvers.js]: If you don't plan to update a variable, make it const.

File: src/schema/resolvers.js [hunk 3]
```diff
@@ -172,6 +172,35 @@ module.exports = {
...
+       let json = await resp.json();
```
Comment (chrisli30) [inline src/schema/resolvers.js]: const json

File: src/schema/resolvers.js (hunk 1 above)
Comment (frankobe) [reply src/schema/resolvers.js]: @chrisli30 dunt think it is necessary. No linter style requires it

File: src/schema/resolvers.js (hunk 2 above)
Comment (frankobe) [reply src/schema/resolvers.js]: Correct me if I am wrong, I take `const` more like a variable for re-use and highlighted. `let` is block scoped so it is cheaper than `var` to use it in small bracket like this

File: src/schema/resolvers.js (hunk 3 above)
Comment (frankobe) [reply src/schema/resolvers.js]: @chrisli30  same as above

File: src/schema/resolvers.js (hunk 2 above)
Comment (chrisli30) [reply src/schema/resolvers.js]: 'let' is the old 'var' in es6 syntax. 'const' disallow a var to be reassigned. The choice is based on intention of the variable. You can won't fix this if you don't want to change.

**Pull request merged**